torch/utils/inference/run.py

```diff
@@ -0,0 +1,569 @@
+r"""
+This is a script for launching PyTorch inference on Intel(R) Xeon(R) Scalable Processors with optimal configurations.
+Single instance inference, multi-instance inference are enabled.
+
+To get the peak performance on Intel(R) Xeon(R) Scalable Processors, the script optimizes the configuration of thread and memory
+management. For thread management, the script configures thread affinity and the preload of Intel OMP library.
+For memory management, it configures NUMA binding and preload optimized memory allocation library (e.g. tcmalloc, jemalloc).
+
```

Comment:
could you also briefly discuss the auto omp settings and the various other environment variable this launcher auto-sets on behalf of the user, it could be as simple as a list of env vars that this launcher may set depending on the options and a brief description of what it does (a link to a more detailed doc is also acceptable)

Reply:
done